### ISSUE TITLE
actionBarHidden seems to have broken in 0.3.0 release

### ISSUE THREAD

**Issue by irthos** [rigor789/nativescript-vue#93]
Heya! Thanks for staying on top of dev here. Super stoked to see and play with the progress!

I just installed the latest version and the action bar won't hide anymore with the code that worked on 0.1.* This is on genymotion emulator android 7.1.0, haven't tried hardware. iOS is good for the initial view but the action bar appears on any route change.

Here's how I had it on the App.vue component. Tried using kebab-case but no diff. Can you recommend another way?
``` 
<template>
    <page class="background" actionBarHidden="true" backgroundSpanUnderStatusBar="false">
        <stack-layout height="100%">
            <router-view></router-view>
        </stack-layout>
    </page>
</template>
```

Also tried the advice of @tralves to do:
```
<template>
    <page ref="page" class="background" backgroundSpanUnderStatusBar="false">
        <stack-layout height="100%">
            <router-view></router-view>
        </stack-layout>
    </page>
</template>

<script>
    import TestComponent from './TestComponent.vue'
    import Auth from './Auth.vue'

    export default {

        data() {
            return {
                count: 10
            }
        },

        components: {
            TestComponent, Auth
        },
        mounted () {
            this.$nextTick(function(){
                this.$refs.page.nativeView.actionBarHidden = true;
            })
        }
    }
</script>
```
That didn't do the trick. Could this have come about while tinkering to get the back button to behave?

**Comment by jlooper** [comment]
I just tested this, and I think it's right, I can't seem to manipulate the ActionBar at all - for example to hide it on a login screen.

**Comment by rigor789** [comment]
@irthos I can't reproduce the issue with a simple case: https://play.nativescript.org/?template=play-vue&id=Br5JbX

Although I suspect it has to do with the routing being broken right now. 

I'm closing this in favor of #98 if we fix #98 and the issue still persist, I'll reopen the issue.

**Comment by irthos** [comment]
Ok sounds good
